source/slang/slang-lower-to-ir.cpp

```diff
@@ -3546,6 +3556,27 @@ struct ExprLoweringContext
         // TODO: also need to handle this-type substitution here?
     }
 
+    void validateInvokeExprArgsWithFunctionModifiers(
```

Comment:
If this modifier is only intend to be used inside stdlib and we know it is never going to be wrong, then we don't need to validate it. This will slow down the compiler unnecessarily.